packages/block-editor/src/components/use-editor-feature/index.js

```diff
@@ -39,16 +39,24 @@ export default function useEditorFeature( featurePath ) {
 	const setting = useSelect(
 		( select ) => {
 			const path = `__experimentalFeatures.${ featurePath }`;
-			const { getSettings } = select( 'core/block-editor' );
-			const settings = getSettings();
 
+			// 1 - Use block.json, if available.
+			const { getBlockSupport } = select( 'core/blocks' );
+			const blockSupportValue = getBlockSupport( blockName, path );
+			if ( blockSupportValue !== undefined ) {
+				return blockSupportValue;
+			}
```

Comment:
Why do we need to check block support here? I think we should remove that check entirely from here. the useEditorFeature should be just about theme.json and block-editor setting and the hook should check for block support. At least that's my reasoning for the change.

Reply:
Actually, I think the issue is on Core's `experimental-theme.json`. It should be "true" there and not disabled. This fil e corresponds to the default value for Core if I'm not wrong.

Reply:
Alternative at https://github.com/WordPress/gutenberg/pull/24932